Mathlib/Data/Fintype/List.lean

```diff
@@ -16,47 +17,92 @@ admits a `Fintype` instance.
 
 ## Implementation details
 To construct the `Fintype` instance, a function lifting a `Multiset α`
-to the `Finset (List α)` that can construct it is provided.
+to the `Multiset (List α)` that can construct it is provided.
 This function is applied to the `Finset.powerset` of `Finset.univ`.
 
-In general, a `DecidableEq` instance is not necessary to define this function,
-but a proof of `(List.permutations l).nodup` is required to avoid it,
-which is a TODO.
-
 -/
 
 
-variable {α : Type*} [DecidableEq α]
-
+variable {α : Type*}
 open List
 
 namespace Multiset
 
-/-- The `Finset` of `l : List α` that, given `m : Multiset α`, have the property `⟦l⟧ = m`.
+/-- The `Multiset` of `l : List α` that, given `m : Multiset α`, have the property `⟦l⟧ = m`.
```

Comment:
Doc-string would be more readable if `m : Multiset α` is mentioned at the beginning of the sentence!

Reply:
moved it to the front